### PR TITLE
[TP][OPS-11132]

### PR DESCRIPTION
**Pull request by tp10**

### CODE REVIEW

File: app/language/Messages.scala [hunk 1]
```diff
@@ -247,9 +247,17 @@ object Messages {
       english = "What happens next"
     )
 
+    val `What do you think`: Message = Message(
+      english = "What did you think of this service ?"
```
Comment (DarrenCB) [inline app/language/Messages.scala:251]: Don't think we need a space before the ?

**Review by DarrenCB — DISMISSED** [review]

File: app/language/Messages.scala (hunk 1 above)
Comment (tp10) [reply app/language/Messages.scala]: Yip, good spot. I've added the copy to the test so pushing up again.

**Review by DarrenCB — APPROVED** [review]

File: app/language/Messages.scala [hunk 2]
```diff
@@ -247,9 +247,17 @@ object Messages {
       english = "What happens next"
     )
 
+    val `What do you think`: Message = Message(
+      english = "What did you think of this service?"
+    )
+
+    val `takes 30 seconds`: Message = Message(
+      english = "takes 30 seconds"
```
Comment (barnesjake) [inline app/language/Messages.scala:255]: should we have the brackets inside the message

**Review by barnesjake — CHANGES_REQUESTED** [review]
Could we have the brackets for "(takes 30 seconds)" inside the messages file, not the twirl template

**Review by barnesjake — APPROVED** [review]

**Review by Lyrain — APPROVED** [review]

**Pull request merged**